packages/web-client/app/components/card-pay/merchant/index.hbs

```diff
@@ -0,0 +1,30 @@
+<div
+  class={{cn "merchant" merchant--vertical=@vertical}}
+  style={{css-var
+    merchant-logo-background=(or @logoBackground "var(--boxel-red)")
+    merchant-logo-text-color=(or @logoTextColor "var(--boxel-light)")
+  }}
+  data-test-merchant={{@name}}
+  data-test-merchant-logo-background={{@logoBackground}}
+  data-test-merchant-logo-text-color={{@logoTextColor}}
+  ...attributes
+>
+  <div class={{cn
+    "merchant__logo"
+    merchant__logo--sm=(eq @size "small")
+    merchant__logo--lg=(eq @size "large")
+  }}>
+    {{truncate @name 1 false}}
```

Comment:
Not a big deal, but this would be more readable if we created a `first-char` helper.